docusaurus.config.js

```diff
@@ -312,6 +312,10 @@ const config = {
                   label: "Rust Docs",
                   href: "/docs/current/developer-docs/backend/rust/?source=nav",
                 },
+                {
+                  label: "Solidity Docs",
+                  href: "https://bitfinity.network/ecosystem/solidity-examples",
```

Comment:
This should link here: https://bitfinity.network/ecosystem/solidity-examples

Reply:
I have updated this.